### PR TITLE
quill:0.1.0

### PR DESCRIPTION
**Pull request by Mc-Zen**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: Quill is a package for creating quantum circuit diagrams in Typst doing a similar job as the much-used LaTeX packages qcircuit and quantikz.

### CODE REVIEW

File: packages/preview/quill-0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Since there is no package manager for Typst yet, in order to use this library, download the [quill.typ](./qcircuit.typ) file and place it in your Typst project.  -->
+
+## Usage
+
+Create circuit diagrams by calling the function `quantum-circuit()` with any number of positional arguments — just like the built-in Typst functions `table()` or `grid()`. A variety of different gate and instructions commands are available and plain numbers can be used to produce any number of empty cells just filled with the current wire mode. A new wire is started by adding the content item `[\ ]`. 
+
+```java
+#{
+  import "quill.typ" : *
```
Comment (laurmaedje) [inline packages/preview/quill-0.1.0/README.md:29]: this import should show how to import as a package, i.e. `import "@preview/quill:0.1.0": *`

File: packages/preview/quill-0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  quantum-circuit(
+    lstick($|0〉$), gate($H$), control(1), rstick($(|00〉+|11〉)/√2$, n: 2), [\ ],
```
Comment (laurmaedje) [inline packages/preview/quill-0.1.0/README.md:31]: I tried this and `control` seems to have been renamed to `ctrl`

**Review by laurmaedje — COMMENTED** [review]
Looks amazing! Two quick fixes to the documentation and we're good to go!

File: packages/preview/quill-0.1.0/README.md (hunk 1 above)
Comment (Mc-Zen) [reply packages/preview/quill-0.1.0/README.md:29]: Yes, I'll change that!

File: packages/preview/quill-0.1.0/README.md (hunk 2 above)
Comment (Mc-Zen) [reply packages/preview/quill-0.1.0/README.md:31]: Dang it! I knew there was still something. Thank you for pointing it out

**Comment by Mc-Zen** [comment]
Okay!

**Comment by laurmaedje** [comment]
Wonderful, the first merged package. 🎉

**Pull request merged**